egs/swbd/run.sh

```diff
@@ -57,6 +57,18 @@ for order in 3 4 5; do
     format_arpa_lm.py data/lm_20k_${order}_prune${threshold} | gzip -c > data/arpa/20k_${order}gram_prune${threshold}.arpa.gz
 
   done
+
+# example of pruning by size.
```

Comment:
I'm a little uncomfortable with having the example script pruning to a proportion like this, because people might think it's a pattern to follow in real-life examples.   In real life ASR applications you'd probably want to prune to a fixed target, like 1.5 million n-grams. 
BTW, make sure your LM-pruning script dies if the target number of n-grams is >= the initial number of n-grams in the LM, or is already within the threshold [in which case there is nothing to do, so best to skip the pruning stage.]